Assets/Mirror/Editor/MirrorAssetProcessor.cs

```diff
@@ -0,0 +1,116 @@
+using UnityEngine;
+using UnityEditor;
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace DigitalRuby.WeatherMaker
+{
+    /// <summary>
+    /// Pre-process define utility
+    /// </summary>
+    public class MirrorAssetPostprocessor : AssetPostprocessor
+    {
+        private const string assetToCheck = "/Mirror/Runtime/NetworkIdentity.cs";
```

Comment:
Anyone has any idea how to make this better?   This will break if users want to move mirror to other folders

Reply:
Absolutely do not make this path dependent.  Mirror folder could be moved, renamed, etc.

Reply:
Wrong namespace

Reply:
Why can't we check for path ends with `MirrorAssetPostprocessor.cs` ?

Reply:
Does it support wildcards? Although depending on the project size. A wildcard scan could take forever.

Reply:
@MrGadget1024 that is a good idea, check for this file itself,  does not matter where it is.  The name is very specific so we are not going to have conflicts with other libs.

@jjxtra please rename MirrorAssetProcessor.cs -> MirrorAssetPostProcessor.cs  to match the name of the class.

Reply:
It is renamed, namespace fixed

Reply:
Could check for ends with, good idea. I'm open to ideas, maybe there is a unique script file we could throw in, it could just be empty...?